public/app/features/logs/components/LogRow.tsx

```diff
@@ -26,7 +23,7 @@ import { LogRowMessage } from './LogRowMessage';
 import { LogRowMessageDisplayedFields } from './LogRowMessageDisplayedFields';
 import { getLogLevelStyles, LogRowStyles } from './getLogRowStyles';
 
-interface Props extends Themeable2 {
+export interface Props {
```

Comment:
if we gonna export this it should have a better name?

Reply:
It's usually exported as Props. https://github.com/search?q=repo%3Agrafana%2Fgrafana+%22export+interface+Props%22&type=code